projects/storefrontlib/shared/components/media/media.service.ts

```diff
@@ -38,7 +38,7 @@ export class MediaService {
    * for specific formats.
    */
   getMedia(
-    mediaContainer: MediaContainer | Image,
+    mediaContainer?: MediaContainer | Image,
```

Comment:
doesn't hurt to make it optional, but do we need to? I understand we can now explicitly pass undefined to the `input container` but I think we can keep it as required for the getMedia function in order for customers to always atleast pass a media. What do you think?

Reason why I'm saying this is ebcause in [line 46](https://github.com/SAP/spartacus/pull/15410/files#diff-b2b4a1ad47f7561a671934b3afed849d7676a290627693fa3c558fe0aeab1e9bR46) we're getting out of the function if it's undefined.